inst/include/staticModelAdapt.h

```diff
@@ -76,5 +76,98 @@ namespace smc {
         /// Returns the empirical covariance matrix based on the current weighted particle set.
         arma::mat GetEmpCov(void) const {return empCov;}
     };
+
```

Comment:
Again, you could debate whether all of what follows belongs inline in headers; much of this is prototypical code that a user would need to replicate to implement a different adaptation scheme, but again I don't feel particularly strongly either way.